### PR TITLE
Remove installing of VLD

### PR DESCRIPTION
**Pull request by dcristoloveanu**
Remove installing of VLD

### CODE REVIEW

**Review by anporumb — APPROVED** [review]
:shipit:

File: build/devops_gated.yml [hunk 1]
```diff
@@ -39,28 +39,12 @@ jobs:
      $wc.DownloadFile($url, $outpath)
     displayName: 'Download VLD'
```
Comment (mattdurak) [inline build/devops_gated.yml:40]: VLD can be removed here too

File: build/devops_gated.yml [hunk 2]
```diff
@@ -180,28 +161,12 @@ jobs:
...
```
Comment (mattdurak) [inline build/devops_gated.yml:162]: and here

**Review by mattdurak — APPROVED** [review]
:shipit:

**Pull request merged**